tests/components/rfxtrx/test_binary_sensor.py

```diff
@@ -69,6 +69,35 @@ async def test_one_pt2262(hass, rfxtrx):
     assert state.state == "off"
 
 
+async def test_pt2262_unconfigured(hass, rfxtrx):
+    """Test with discovery for PT2262."""
+    assert await async_setup_component(
+        hass,
+        "rfxtrx",
+        {
+            "rfxtrx": {
+                "device": "abcd",
+                "devices": {
+                    "0913000022670e013970": {},
+                    "09130000226707013970": {},
+                },
+            }
+        },
+    )
+    await hass.async_block_till_done()
+    await hass.async_start()
+
+    state = hass.states.get("binary_sensor.pt2262_22670e")
+    assert state
+    assert state.state == "off"  # probably aught to be unknown
+    assert state.attributes.get("friendly_name") == "PT2262 22670e"
+
+    state = hass.states.get("binary_sensor.pt2262_226707")
+    assert state
+    assert state.state == "off"  # probably aught to be unknown
+    assert state.attributes.get("friendly_name") == "PT2262 226707"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6090606)